src/lattice/types.ts

```diff
@@ -15,134 +15,154 @@ export const DEFAULT_LATTICE_UNITS = {
  *   Challenges and tools. Computational Materials Science, 49(2), 299-312. doi:10.1016/j.commatsci.2010.05.010
  * Lattice parameters a, b, c are stored for CONVENTIONAL lattice, however the default unit cell is calculated for
  */
-export const LATTICE_TYPE = {
-    CUB: "CUB",
-    FCC: "FCC",
-    BCC: "BCC",
-    TET: "TET",
-    BCT: "BCT",
-    ORC: "ORC",
-    ORCF: "ORCF",
-    ORCI: "ORCI",
-    ORCC: "ORCC",
-    HEX: "HEX",
-    RHL: "RHL",
-    MCL: "MCL",
-    MCLC: "MCLC",
-    TRI: "TRI",
-};
 
-export const LATTICE_TYPE_EXTENDED = {
-    BCC: "BCC",
-    BCT_1: "BCT-1",
-    BCT_2: "BCT-2",
-    CUB: "CUB",
-    FCC: "FCC",
-    HEX: "HEX",
-    MCL: "MCL",
-    MCLC_1: "MCLC-1",
-    MCLC_2: "MCLC-2",
-    MCLC_3: "MCLC-3",
-    MCLC_4: "MCLC-4",
-    MCLC_5: "MCLC-5",
-    ORC: "ORC",
-    ORCC: "ORCC",
-    ORCF_1: "ORCF-1",
-    ORCF_2: "ORCF-2",
-    ORCF_3: "ORCF-3",
-    ORCI: "ORCI",
-    RHL_1: "RHL-1",
-    RHL_2: "RHL-2",
-    TET: "TET",
-    TRI_1a: "TRI_1a",
-    TRI_2a: "TRI_2a",
-    TRI_1b: "TRI_1b",
-};
+export enum LatticeType {
+    CUB = "CUB",
+    FCC = "FCC",
+    BCC = "BCC",
+    TET = "TET",
+    BCT = "BCT",
+    ORC = "ORC",
+    ORCF = "ORCF",
+    ORCI = "ORCI",
+    ORCC = "ORCC",
+    HEX = "HEX",
+    RHL = "RHL",
+    MCL = "MCL",
+    MCLC = "MCLC",
+    TRI = "TRI",
```

Comment:
Any way we could move these enums to ESSE also, to be able to potentially reuse in Python in the future?

Reply:
We already have defined  `LatticeType` in ESSE `LatticeImplicitSchema`, so we can use them from there. 
But I didn't find `LatticeTypeExtended` there. We can define another schema for this

Reply:
Note: we will be using strings instead of enums
https://github.com/bcherny/json-schema-to-typescript/issues/200